src/literal.h

```diff
@@ -612,6 +612,10 @@ class Literal {
   Literal floorF64x2() const;
   Literal truncF64x2() const;
   Literal nearestF64x2() const;
+  Literal extAddPairwiseSToI16x8() const;
+  Literal extAddPairwiseUToI16x8() const;
+  Literal extAddPairwiseSToI32x4() const;
+  Literal extAddPairwiseUToI32x4() const;
```

Comment:
```suggestion
  Literal extAddPairwiseToSI16x8() const;
  Literal extAddPairwiseToUI16x8() const;
  Literal extAddPairwiseToSI32x4() const;
  Literal extAddPairwiseToUI32x4() const;
```
This would be more consistent with the names below. Also, I notice that we aren't consistent about whether the names include "Vec". It would be good to clean that up later.

Reply:
> It would be good to clean that up later.

Yeah, I guess Vec is redundant due to we already have MxN prefixes